examples/wordcount/docker/wordcount/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+ADD . /code
+
+RUN apt-get update
+RUN echo "/usr/local/lib" >> /etc/ld.so.conf
+RUN git clone https://github.com/edenhill/librdkafka.git /tmp/librdkafka
+RUN ls /tmp/ && cd /tmp/librdkafka && ./configure && make && make install && ldconfig
+
+WORKDIR /code/examples/wordcount/
+RUN pip --version
+#RUN pip install -e git+https://github.com/confluentinc/confluent-kafka-python.git#egg=confluent-kafka
```

Comment:
Probably the pip install below should pull this in, do you remember why you have the commented out pip install confluent-kafka-python here?

Getting this:
```
$ docker logs docker_wordcount_1
Traceback (most recent call last):
  File "example.py", line 13, in <module>
    from winton_kafka_streams.processor import BaseProcessor, TopologyBuilder
  File "/code/winton_kafka_streams/processor/__init__.py", line 10, in <module>
    from ._stream_thread import StreamThread
  File "/code/winton_kafka_streams/processor/_stream_thread.py", line 10, in <module>
    from confluent_kafka import KafkaError
ModuleNotFoundError: No module named 'confluent_kafka'
```